packages/nimble-components/src/text-area/styles.ts

```diff
@@ -54,7 +67,8 @@ export const styles = css`
         align-items: flex-end;
         border: ${borderWidth} solid transparent;
         padding: 8px;
-        transition: box-shadow ${smallDelay}, border ${smallDelay};
+        transition: border ${smallDelay};
+        margin-bottom: 1px;
```

Comment:
As part of just highlighting the bottom border, we now thicken it by changing the bottom border width, rather than using a box shadow. Because of this, the thickening actually affects layout (by one pixel). To keep things from shifting, we put a 1px margin on the control, then remove it when thickening the border.